sandbox/anvi-script-gen-genomes-file

```diff
@@ -0,0 +1,136 @@
+
+        contig_db_path = os.path.abspath(args.contigs_db)
+        profile_db_path = os.path.abspath(args.profile_db)
+
+        int_genomes_dict = {}
+        for bin in sorted(collections.collections_dict[args.collection_name]['bin_names'].split(',')):
+            int_genomes_dict[bin] = {}
+            int_genomes_dict[bin]['bin_id'] = bin
+            int_genomes_dict[bin]['collection_id'] = args.collection_name
+            int_genomes_dict[bin]['profile_db_path'] = profile_db_path
+            int_genomes_dict[bin]['contigs_db_path'] = contig_db_path
+
+        utils.store_dict_as_TAB_delimited_file(int_genomes_dict, args.internal_genomes, key_header='name')
+
+        run.info("Internal genomes file", args.internal_genomes)
+
+    elif args.input_dir:
+        if not args.external_genomes:
+            raise ConfigError("Please provide an output file path for your external genomes file using the -e parameter.")
+
+        if not filesnpaths.is_file_exists(args.input_dir, dont_raise=True):
+            raise ConfigError("The directory you provided does not exist :/")
+
+        if filesnpaths.is_file_exists(args.external_genomes, dont_raise=True):
+            raise ConfigError(f"The external genomes file path that you provided already exists, and anvi'o will not overwrite "
+                              f"it. Please either remove the existing file or provide a different file path.")
+
+        ext_genomes_dict = {}
+        for file in glob.glob(os.path.join(args.input_dir, '*.db')):
+            if utils.is_contigs_db(file, dont_raise=True):
+                db_name = os.path.basename(file).split(".db")[0]
+                db_path = os.path.abspath(file)
+                ext_genomes_dict[db_name] = {}
+                ext_genomes_dict[db_name]['contigs_db_path'] = db_path
+
+        if not ext_genomes_dict:
+            raise ConfigError("No contigs databases were found in the provided input directory.")
+
+        utils.store_dict_as_TAB_delimited_file(ext_genomes_dict, args.external_genomes, key_header='name')
+
+        run.info("External genomes file", args.external_genomes)
+
+    else:
+        raise ConfigError("You must provide either an input directory (for an external genomes file) or a "
+                          "profile db/contigs db/collection name set (for an internal genomes file). Also, "
+                          "just so you don't forget, we also need an output file path in either case.")
+
+if __name__ == '__main__':
+    from anvio.argparse import ArgumentParser
+
+    parser = ArgumentParser(description=__description__)
+
+    groupE = parser.add_argument_group("EXTERNAL GENOMES FILE", "Provide a directory, and anvi'o will provide an external genomes "
+                                                                "file containing all contigs dbs in that directory.")
+    groupE.add_argument(*['-I', '--input-dir'], **anvio.K('input-dir'))
```

Comment:
`-I` is used for internal HMM models :) I would remove it and go with `*anvio.A('input-dir')` here.

Reply:
I started out with `*anvio.A('input-dir')`, but unfortunately that one takes an `-i` flag which conflicts with the `-i` for `*anvio.A('internal-genomes')` :) 
But what I can do is change the `-I` flag to something else, maybe `-D`? Or I can get rid of the short flag altogether and force people to use `--input-dir` in full.

Reply:
Let's remove `-i` for `--input-dir` from init since it shouldn't have been there at the first place.